analysers/commons/commons.py

```diff
@@ -128,13 +138,39 @@ def sortAndGet(CassandraRDD, field, asc):
         .collect()
     
     # TODO: Fix this
-    return {"median_min":medianMin, "median_max":medianMax, \
-              "mean_min":meanMin, "mean_max":meanMax, \
+    return {"mean_min":meanMin, "mean_max":meanMax, \
               "min":dataMin, "max":dataMax, "q1_min":q1Min, \
               "q1_max":q1Max, "q2_min":q2Min, "q2_max":q2Max, \
+              "p90_max":p90Max, "p90_min":p90Min, \
               "p95_max":p95Max, "p95_min":p95Min, \
+              "p99_max":p99Max, "p99_min":p99Min, \
               "q3_min":q3Min, "q3_max":q3Max, "weighted_avg":weightedMean, \
-              "best": bestTrials, "worst": worstTrials, "average": averageTrials}
+              "best": bestTrials, "worst": worstTrials, "average": averageTrials, \
+              "variation_coefficient": coefficientOfVariation}
+
+def computeLevene(sc, cassandraKeyspace, expTable, dataTable, experimentID, containerID, hostID, dataName):
+    try:
+        trials = sc.cassandraTable(cassandraKeyspace, expTable) \
+            .select("trial_id") \
+            .where("experiment_id=?", experimentID) \
+            .map(lambda a: a["trial_id"]) \
+            .distinct() \
+            .collect()
+        samples = []
+        for trial in trials:
```

Comment:
If you have at least two samples, you can perform the test. Check at least that before applying the computation